templates/frontend/objects/article_details.tpl

```diff
@@ -396,6 +396,14 @@
 				</div>
 			{/if}
 
+			{* Data Availability Statement *}
+			{if $publication->getLocalizedData('dataAvailability')}
+				<section class="item dataAvailability">
+					<h2 class="label">{translate key="submission.dataAvailability"}</h2>
+					{$publication->getLocalizedData('dataAvailability')|strip_unsafe_html}
```

Comment:
The data availability statement doesn't appear to be a rich text field, but it's being escaped here with `strip_unsafe_html`, which suggests it should be. Either:
- Choose HTML, in which case the input field should be a rich field, or
- Choose plain text, in which case the value should be passed through `|escape` for display.

(This may affect the other application PRs, if plain text is chosen.)